### PR TITLE
[IMP] _adjust_merged_values_orm: add first_not_null option for chars

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
:)

### CODE REVIEW

File: openupgradelib/openupgrade_merge_records.py [hunk 1]
```diff
@@ -365,6 +366,10 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
         if field.type in ('char', 'text', 'html'):
             if not op:
                 op = 'other' if field.type == 'char' else 'merge'
+            if op == 'first_not_null':
+                _list = filter(lambda x: x is not False, _list)
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade_merge_records.py:370]: Is it possible to have `""` instead of False?

Comment (MiquelRForgeFlow) [reply openupgradelib/openupgrade_merge_records.py]: Yeah, `""` cases are excluded now.

File: openupgradelib/openupgrade_merge_records.py [hunk 2]
```diff
@@ -365,6 +366,10 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
         if field.type in ('char', 'text', 'html'):
             if not op:
                 op = 'other' if field.type == 'char' else 'merge'
+            if op == 'first_not_null':
+                _list = filter(lambda x: x, _list)
+                if _list:
+                    vals[field.name] = _list[0]
             if op == 'merge':
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade_merge_records.py:373]: Just for a little code performance optimization
```suggestion
            elif op == 'merge':
```

Comment (MiquelRForgeFlow) [reply openupgradelib/openupgrade_merge_records.py]: done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**